src/cipher/mod.rs

```diff
@@ -16,9 +16,6 @@ pub const CIPHER_SIZE: usize = MESSAGE_CAPACITY + 1;
 /// Number of bytes used by from/to bytes `PoseidonCipher` function
 pub const CIPHER_BYTES_SIZE: usize = CIPHER_SIZE * 32;
 
-/// Bytes consumed on serialization of the poseidon cipher
-pub const ENCRYPTED_DATA_SIZE: usize = CIPHER_SIZE * 32;
```

Comment:
Don't remove this. I believe is still used across other repos since it was a Public API. You can add a TODO note in order to unify with the `CIPHER_BYTES_SIZE` or something like that. I believe that `ENCRYPTED_DATA_SIZE` is used more across our repos than `CIPHER_BYTES_SIZE`, also.

Reply:
Done